Source/santad/testdata/protobuf/v1/exec.json

```diff
@@ -120,5 +120,43 @@
   }
  },
  "explain": "extra!",
- "quarantine_url": "google.com"
+ "quarantine_url": "google.com",
+ "entitlements": [
+  {
+   "key": "key_with_arr_val",
+   "value": "[\"v1\",\"v2\",\"v3\"]"
+  },
+  {
+   "key": "key_with_data_val",
+   "value": "\"SGVsbG8gV29ybGQ=\""
+  },
+  {
+   "key": "key_with_arr_val_multitype",
+   "value": "\"(\\n    v1,\\n    v2,\\n    v3,\\n    123,\\n    \\\"2023-11-07 17:00:02 +0000\\\"\\n)\""
```

Comment:
Sorta related to date formatting q above: this is a different format than the `key_with_date_val` below. I know nested values aren't common so it's not the primary concern, but seeing different formats is confusing...

Reply:
See other comment - will make things consistent by first standardizing types in nested objects.